### DISCUSSION TITLE
Game controller support?

### DISCUSSION THREAD

**Discussion by jeffnguyener** [Whisky-App/Whisky#254] [Q&A]
Tried to use 8bitdo Pro controller and it doesn't work. Does anyone have a resolution or a way to enable it? I tried to play around with the Game Controllers in the Control Panel already.

**Comment by Mateus109** [comment]
My PS4 controller works perfectly.  So it's odd that the 8bitdo Pro doesn't.

**Comment by Crd5** [comment]
Does it work in the Game Controllers in the Control Panel?

**Reply by jeffnguyener** [reply]
It shows up on the Game Controller panel as a connected (xinput device) but doesn't do anything when testing joystick. I also overrided it to be in the "connected" section and does the same thing.

**Comment by Jayian1890** [comment]
I have this exact issue with my 8bitdo controller. If you connect it using the xbox option(not sure if yours has that). It works. Albeit with messed up button mappings.
